src/Symfony/Component/Serializer/Exception/InvalidArgumentException.php

```diff
@@ -16,6 +16,6 @@
  *
  * @author Johannes M. Schmitt <schmittjoh@gmail.com>
  */
-class InvalidArgumentException extends \InvalidArgumentException implements ExceptionInterface
+class InvalidArgumentException extends \InvalidArgumentException implements ExceptionInterface, AggregableExceptionInterface
```

Comment:
After a quick look I think there is two spots that should not be handled:
https://github.com/symfony/symfony/blob/49eafee03323253328d76ec9cb4c515f94d2f202/src/Symfony/Component/Serializer/Normalizer/DataUriNormalizer.php#L102
https://github.com/symfony/symfony/blob/49eafee03323253328d76ec9cb4c515f94d2f202/src/Symfony/Component/Serializer/Normalizer/ArrayDenormalizer.php#L48

Those cases are because of an invalid type or missing class, I think they should stop the denormalization process as usual.